commit b8c6166b1b75036ab3e4127a1c0aacf52ca93651
Author: Florian Weimer <fweimer@redhat.com>
Date:   Mon Sep 6 19:43:37 2021 +0200

    locale: Add missing second argument to _Static_assert in C-collate-seq.c

diff --git a/locale/C-collate-seq.c b/locale/C-collate-seq.c
index 4fb82cb8357936b6..40b17f4f90313144 100644
--- a/locale/C-collate-seq.c
+++ b/locale/C-collate-seq.c
@@ -55,7 +55,7 @@ static const char collseqmb[] =
 
 /* This table must be 256 bytes in size. We index bytes into the
    table to find the collation sequence.  */
-_Static_assert (sizeof (collseqmb) == 256);
+_Static_assert (sizeof (collseqmb) == 256, "sizeof (collseqmb)");
 
 static const uint32_t collseqwc[] =
 {
